drivers/gles3/storage/texture_storage.cpp

```diff
@@ -1495,14 +1495,17 @@ void TextureStorage::_texture_set_data(RID p_texture, const Ref<Image> &p_image,
 	for (int i = 0; i < mipmaps; i++) {
 		int size, ofs;
 		img->get_mipmap_offset_and_size(i, ofs, size);
-
 		if (compressed) {
-			glPixelStorei(GL_UNPACK_ALIGNMENT, 4);
```

Comment:
This line should not have been deleted. We should be using an alignment of 4 for compressed textures

Reply:
4 is a default, but better to explicitly define, aggree. Deleted at some point and havent returned. sorry. rolled back.